lib/curl/curl.nit

```diff
@@ -246,6 +249,116 @@ class CurlHTTPRequest
 	end
 end
 
+
+# Restful JSON request builder over HTTP
+#
+class CurlRestRequest
+	super CurlHTTPRequest
+	var method : nullable String
```

Comment:
what if `method` remains null?
either test the null case and assume it means GET, or use a default value with something like

~~~nit
var method: String = "GET" is optional, writable
~~~

Reply:
Libcurl's default behavior is to send a GET request. No need to ensure its set. I assume the user will RTFM.